README.md

```
@@ -39,7 +39,24 @@ It also provides support for shard aware ports, a faster way to connect to all s
   - [5.1 Shard-aware port](#51-shard-aware-port)
   - [5.2 Client routes (PrivateLink)](#52-client-routes-privatelink)
   - [5.3 Iterator](#53-iterator)
-- [6. Contributing](#6-contributing)
+- [6. Redis Compatibility Mode](#6-redis-compatibility-mode)
+  - [6.1 Basic Usage (Set/Get/MGet/MSet/Del/Exists)](#61-basic-usage-setgetmgetmsetdelexists)
+  - [6.1.1 Migration from go-redis](#611-migration-from-go-redis)
+  - [6.2 SetNX / SetEx / GetSet / GetDel](#62-setnx--setex--getset--getdel)
+  - [6.3 Counters (Incr/Decr)](#63-counters-incrdecr)
+  - [6.4 Key Lifecycle (Expire/TTL/Persist)](#64-key-lifecycle-expireTTLpersist)
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid
Expected: `#64-key-lifecycle-expirettlpersist`; Actual: `#64-key-lifecycle-expireTTLpersist`

(MD051, link-fragments)